### PR TITLE
chore: tag graalvm test images with infrastructure-public-image- suffix

### PR DESCRIPTION
**Pull request by mpeddada1**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-shared-config/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

If you write sample code, please follow the [samples format](
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md).

### CODE REVIEW

**Comment by mpeddada1** [comment]
Confirmation of tagged images:

Screenshot 2024-05-23 at 6 02 39 PM

File: .cloudbuild/cloudbuild-test-b.yaml [hunk 1]
```diff
@@ -19,7 +19,9 @@ substitutions:
 steps:
   # GraalVM B build
   - name: gcr.io/cloud-builders/docker
-    args: ["build", "-t", "gcr.io/cloud-devrel-public-resources/graalvm_b:${_JAVA_SHARED_CONFIG_VERSION}", "--file", "graalvm-b.Dockerfile", "."]
```
Comment (suztomo) [inline .cloudbuild/cloudbuild-test-b.yaml:22]: This file is for testing, right? Then it's not needed.

File: .cloudbuild/cloudbuild-test-a.yaml [hunk 2]
```diff
@@ -19,7 +19,9 @@ substitutions:
 steps:
   # GraalVM A build
   - name: gcr.io/cloud-builders/docker
-    args: ["build", "-t", "gcr.io/cloud-devrel-public-resources/graalvm_a:${_JAVA_SHARED_CONFIG_VERSION}", "--file", "graalvm-a.Dockerfile", "."]
```
Comment (suztomo) [inline .cloudbuild/cloudbuild-test-a.yaml:22]: This file is for testing, right? Then it's not needed.

**Review by suztomo — APPROVED** [review]

File: .cloudbuild/cloudbuild-test-b.yaml (hunk 1 above)
Comment (mpeddada1) [reply .cloudbuild/cloudbuild-test-b.yaml]: Yup this is only for testing, done.

**Pull request merged**